justfile

```diff
@@ -0,0 +1,90 @@
+targets := "arm-unknown-linux-gnueabi armv7-unknown-linux-gnueabihf x86_64-unknown-linux-gnu x86_64-unknown-linux-musl thumbv6m-none-eabi thumbv7em-none-eabi thumbv7em-none-eabihf thumbv7m-none-eabi"
```

Comment:
The suggestions for `tinytga` also apply to this justfile, except for the comments that contain `tinytga`.